temba/externals/views.py

```diff
@@ -77,3 +81,23 @@ class Delete(DependencyDeleteModal):
         cancel_url = "@orgs.org_home"
         success_url = "@orgs.org_home"
         success_message = _("Your external service has been deleted.")
+
+
+class GetExternalTypes(APIView):
+    authentication_classes = []
+    permission_classes = []
+    pagination_class = None
+    renderer_classes = [JSONRenderer]
+    throttle_classes = []
+
+    def get(self, request, slug=None):
+        from temba.externals.types import TYPES
+
+        external_type = TYPES.get(slug)
+        
+        if external_type is None:
+            return Response (status=status.HTTP_404_NOT_FOUND)
```

Comment:
on line 99, there is white space before '('
`return Response (status=status.HTTP_404_NOT_FOUND)` 
try:
`return Response(status=status.HTTP_404_NOT_FOUND)`